code/modules/mob/living/carbon/human/species/station/human_subspecies.dm

```diff
@@ -37,9 +37,14 @@
 	name_plural = "Vatborn"
 	blurb = "With cloning on the forefront of human scientific advancement, cheap mass production \
 	of bodies is a very real and rather ethically grey industry. Vat-grown or Vatborn humans tend to be \
-	paler than baseline, with no appendix and fewer inherited genetic disabilities, but a more aggressive metabolism."
+	paler than baseline, with no appendix and fewer inherited genetic disabilities, but a more aggressive metabolism. \
+	Most vatborn are engineered to experience rapid maturation, reaching approximately sixteen years of growth in only five \
```

Comment:
[The wiki](https://wiki.ss13polaris.com/index.php?title=Vatborn#.22Natural.22_Vatborn) seems to contradict this claim of "Most".
> The most common form of Vatborn are grown only as infants and "born" in a similar state to standard human babies; this allows the individuals to 'naturally' learn basic human abilities such as body coordination and speech skills as any other child as the brain develops.

> Fully-grown "Expedited" or "Adult" Vatborn are produced in a process typically reserved for cloning. [...] Illegal clone farm operations in violation of the Five Points have been recorded

Reply:
> With the exception of "designer babies" ordered by wealthy parents seeking an artificial surrogate, most are engineered to undergo rapid maturation, reaching the equivalent of sixteen years of growth in just five.